programs/piaware/faup1090.tcl

```diff
@@ -406,7 +406,7 @@ proc stop_faup1090_close_faup1090_socket_and_reopen {} {
 # attempt_dump1090_restart - restart dump1090 if we can figure out how to
 #
 proc attempt_dump1090_restart {{action restart}} {
-	set scripts [glob -nocomplain /etc/init.d/*dump1090*.sh]
+	set scripts [glob -nocomplain /etc/init.d/*dump1090*]
```

Comment:
This is possibly a regression (I seem to remember a problem with the more general glob picking up e.g. .dpkg-old files)